pkg/sabakan-cryptsetup/cmd/driver.go

```diff
@@ -152,7 +156,18 @@ func (d *Driver) setupDisk(ctx context.Context, disk Disk, tpmKek []byte) error
 		})
 		return d.formatDisk(ctx, disk, f, tpmKek)
 	}
-	return err
+
+	log.Error("failed to retrieve key from sabakan", map[string]interface{}{
+		log.FnError: err,
+		"disk":      disk.Name(),
+		"try":       retries + 1,
+	})
+	if retries == maxRetry {
+		return err
+	}
+	retries++
+	time.Sleep(time.Duration(retries) * time.Second * 2)
```

Comment:
Just to confirm, do you intend to make sleep time longer if retry happens multiple times?

Reply:
Yes.